src/renderer/components/ft-list-video/ft-list-video.vue

```diff
@@ -72,7 +72,7 @@
         <ft-icon-button
           v-if="inUserPlaylist && canMoveVideoUp"
           :title="$t('User Playlists.Move Video Up')"
-          :icon="['fas', 'arrow-up']"
+          :icon="((listType === 'list' || forceListType === 'list') && forceListType !== 'grid') ? ['fas', 'arrow-up'] : ['fas', 'arrow-left']"
```

Comment:
Maybe make the condition a computed property since it's used twice? `((listType === 'list' || forceListType === 'list') && forceListType !== 'grid')`

Reply:
Five times, actually! (Good call-out)